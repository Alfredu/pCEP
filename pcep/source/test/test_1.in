detect1.dolce
1 amb_temp float Id 44.45 string R "asd" float V 44.567 string foi "foi1"
