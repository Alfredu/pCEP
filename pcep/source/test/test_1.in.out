2 average_temp float fftval 364130.687500 string foi "foi1"
7 Shutdown int Result 0 
